mod/APRandomizer.cs

```diff
@@ -331,6 +331,12 @@ private void Start()
             Victory.OnCompleteSceneLoad(scene, loadScene);
         };
 
+        // update the Nomai text setting before any can be created
+        LoadManager.OnStartSceneLoad += (scene, loadScene) =>
+        {
+            NomaiTextQoL.AutoNomaiText = AutoNomaiText;
+        };
```

Comment:
It has to be _start_ scene load, not the usual complete scene load?

Reply:
Just want to be absolutely sure that no Nomai Wall Text exists yet.

Reply:
Going to leave this as is unless you really don't like it being here.